airbyte-ci/connectors/pipelines/pipelines/dagger/actions/secrets.py

```diff
@@ -42,9 +41,12 @@ async def download(context: ConnectorContext, gcp_gsm_env_variable_name: str = "
     Returns:
         Directory: A directory with the downloaded secrets.
     """
+    # temp - fix circular import
```

Comment:
@bnchrch this is the one circular import I just couldn't figure out how to implement in a decent way. It's also imported [here](https://github.com/airbytehq/airbyte/pull/31525/files#diff-b84200253d5dee854ad304834122ab7eb89978982d6d081a58337b9cf1dd03efR83). Maybe you can take a crack at it